util/libcrypto.num

```diff
@@ -4748,8 +4748,8 @@ EVP_PKEY_verify_init_ex                 4864	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_verify_recover_init_ex         4865	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_CTX_get_signature_md           4866	3_0_0	EXIST::FUNCTION:
 EVP_PKEY_CTX_get_params                 4867	3_0_0	EXIST::FUNCTION:
-EVP_PKEY_CTX_gettable_params            4868	3_0_0	EXIST::FUNCTION:
-EVP_PKEY_CTX_settable_params            4869	3_0_0	EXIST::FUNCTION:
+EVP_PKEY_gettable_ctx_params            4868	3_0_0	EXIST::FUNCTION:
+EVP_PKEY_settable_ctx_params            4869	3_0_0	EXIST::FUNCTION:
```

Comment:
Side note: this one needs more investigation...

Reply:
It shouldn't have been renamed.  It takes a PKEY_CTX argument.
Fix pushed.